### ISSUE TITLE
Dependency not available in ppc64le

### ISSUE THREAD

**Issue by blink1073** [conda-forge/notebook-feedstock#72]
See https://travis-ci.com/github/conda-forge/notebook-feedstock/jobs/367192817 from https://github.com/conda-forge/notebook-feedstock/pull/71.   `argon2-cffi` is not yet available.

**Comment by jjhelmus** [comment]
conda-forge/conda-forge-pinning-feedstock#716 add `argon2-cffi` to the arch migration.  Once merged the bot will automatically start the process of building `argon2-cffi` and its dependencies for `linux-ppc64le` and `linux-aarch64`.

**Comment by blink1073** [comment]
Thanks!

**Comment by bollwyvl** [comment]
Probably also need to request pypy, but I've found it best to just have one open migration at a time to avoid crazy merge conflicts...

**Comment by bollwyvl** [comment]
Looks like the is already there, with the double py36 being pypy. Probably good to rekick in the next hour, once everything's on CDN!

https://anaconda.org/conda-forge/argon2-cffi/files?version=20.1.0

```
linux-aarch64/argon2-cffi-20.1.0-py36h8c4c3a4_0.tar.bz2
linux-aarch64/argon2-cffi-20.1.0-py36h97a6639_0.tar.bz2
linux-aarch64/argon2-cffi-20.1.0-py37h8f50634_0.tar.bz2 
linux-aarch64/argon2-cffi-20.1.0-py38h1e0a361_0.tar.bz2
linux-ppc64le/argon2-cffi-20.1.0-py36h2254977_0.tar.bz2
linux-ppc64le/argon2-cffi-20.1.0-py36h4d54467_0.tar.bz2
linux-ppc64le/argon2-cffi-20.1.0-py37h2bd1440_0.tar.bz2
linux-ppc64le/argon2-cffi-20.1.0-py38h8e95b53_0.tar.bz2
```

**Comment by bollwyvl** [comment]
Fixed by conda-forge/argon2_cffi-feedstock#22
